buildscripts/semantic-convention/templates/ResourceAttributes.java.j2

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Can't we use the same template for both?

Reply:
at the moment, there's a hand-added one at the end of the other template, so it would require some better jinja hacking than I'm currently qualified for.

Reply:
I figured it out. easier than I thought. updated, thanks!